service/tbc/tbc.go

```diff
@@ -1030,14 +1203,10 @@ func (s *Server) syncBlocks(ctx context.Context) {
 			log.Errorf("sync blocks best block header: %v", err)
 			return
 		}
-		s.quiesced = true // XXX if it's set and we exit with an error, what should we do??
 		go func() {
-			// we really want to push the indexing reentrancy into this call
-			log.Infof("quiescing p2p and indexing to: %v @ %v",
-				bhb, bhb.Height)
 			if err = s.SyncIndexersToHash(ctx, bhb.BlockHash()); err != nil {
-				log.Errorf("sync blocks: %v", err)
-				return
+				// XXX this a panic?
+				panic(fmt.Errorf("sync blocks: %v", err))
```

Comment:
This is a panic, so it doesn't matter much, but staticcheck may complain:
```suggestion
				panic(fmt.Errorf("sync blocks: %w", err))
```